docs/general/installation/container.md

```diff
@@ -311,28 +311,21 @@ SuccessExitStatus=0 143
 WantedBy=default.target
 ```
 
-## TrueNAS SCALE / TrueCharts
+## TrueNAS SCALE
 
-Jellyfin is available as a [TrueNAS SCALE](https://www.truenas.org/) App inside the [TrueCharts](https://www.truecharts.org/) App Catalog with direct integration into the GUI, no CLI needed. Direct support is available on the [TrueCharts Discord](https://discord.gg/tVsPTHWTtr) and the source code is available on [GitHub](https://github.com/truecharts/charts).
+Jellyfin is available as a [TrueNAS SCALE](https://www.truenas.org/) App in the official app catalog with direct integration into the GUI, no CLI needed.
 
-1. Install the TrueCharts Catalog to TrueNAS SCALE, see [website](https://truecharts.org/manual/SCALE/guides/getting-started/#adding-truecharts) for more info.
+1. Go to Apps page from the top level SCALE menu
 
-   1. Go to Apps page from the top level SCALE menu
-   2. Select Manage Catalogs tab on the Apps page
-   3. Click Add Catalog
-   4. After reading the iXsystems notice, click Continue and enter the required information:
-      Name: truecharts
-      Repository: `https://github.com/truecharts/catalog`
-      Preferred Trains: `enterprise` and `stable`
-      Branch: main
-   5. Click Save and allow SCALE to refresh its catalog with TrueCharts (this may take a few minutes)
+2. Click `Discover Apps` and search for `Jellyfin`
 
-2. Click `Available Applications` and search for `Jellyfin`
+3. Click `Install`, which will take you to the GUI installation wizard and you'll be able to fill out the necessary info
 
-3. Click `Install`, which will take you to the GUI Wizard and you'll be able to fill out the necessary info
+   - Published Server URL to publish in UDP Auto Discovery response
+   - User and Group Configuration, Network Configuration
+   - Storage Configuraton
+     - For Jellyfin Config Storage and Jellyfin Cache Storage, consider mapping Host Paths for ease of backup
```

Comment:
A few things.
1. Please specify that this SHOULD be on SSDs and not HDDs. Putting this on HDDs WILL lead to a poor experience 
2. It is generally not advised to store databases on ZFS or other CoW file systems but this is Truenas Scale so whatever

Reply:
1. Done
2. Welp, there's no escaping from ZFS on TrueNAS